### PR TITLE
Add --password flag so CCP can retrieve REST API password #66

### PR DESCRIPTION
**Pull request by AndrewCopeland**
- Add `--password` flag so CCP can retrieve REST API password #66

### CODE REVIEW

File: cmd/logon.go [hunk 1]
```diff
@@ -53,6 +59,10 @@ var logonCmd = &cobra.Command{
 			password = string(byteSecretVal)
 		}
 
+		if Password != "" {
+			password = Password
```
Comment (infamousjoeg) [inline cmd/logon.go:63]: The `cybr ccp get-account` command returns the full CCP JSON response blob and doesn't parse the password out to allow command substitution.  We should either provide a flag in `cybr ccp get-account` to return ONLY the password without new-line -OR- figure out logic in `cmd/logon.go` to parse the password out of the full JSON response.

```shell
$ cybr logon -a ldap -b https://cyberark.joegarcia.dev -u jgarcia --non-interactive -p $(cybr ccp get-account -b https://cyberark.joegarcia.dev -i test-cli -s D-Win-SvcAccts -o "Operating System-WinDomain-10.0.4.48-Svc_CybrCLI")
2021/03/29 09:46:54 Failed to Logon to the PVWA. Failed to authenticate to the PAS REST API. Received non-200 status code '403'```

**Review by infamousjoeg — CHANGES_REQUESTED** [review]
I've added requested changes to `cmd/logon.go`.

**Comment by infamousjoeg** [comment]
@AndrewCopeland I've added Issue #103 on to this PR, as well. We can use this for both.

**Comment by AndrewCopeland** [comment]
The `-f` field will parse the result for your. So in the example above you should add `-f Content` so it only returns the `Content` field which is the password for CCP.

**Review by infamousjoeg — APPROVED** [review]
```shell
$ cybr logon -a ldap -b https://cyberark.joegarcia.dev -u Svc_CybrCLI --non-interactive -p $(cybr ccp get-account -b https://cyberark.joegarcia.dev -i test-cli -s D-Win-SvcAccts -o "Operating System-WinDomain-10.0.4.48-Svc_CybrCLI" -f Content)
Successfully logged onto PAS as user Svc_CybrCLI.```

**Pull request merged**